### DISCUSSION TITLE
Contributing to dev branch

### DISCUSSION THREAD

**Discussion by Vishu26** [community/community#18250] [Other Feature Feedback, Questions, & Ideas] [answered]
Whenever I contribute to the dev branch of a repository, I am not listed in the list of contributors. How does the list of contributors work?

**Comment by DavidYang2149** [comment]
Hello!
list of contributors depend on default(usually main) branch.

**Comment by Zi-Tao** [comment, marked as answer]
@Vishu26  When your contributions is merged into the main branch, your contributions will be listed in the contributors list.
